lldb/test/API/lang/swift/foundation_value_types/measurement/TestSwiftFoundationTypeMeasurement.py

```diff
@@ -9,11 +9,41 @@
 # See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
 #
 # ------------------------------------------------------------------------------
-import lldbsuite.test.lldbinline as lldbinline
+import lldb
 from lldbsuite.test.decorators import *
+from lldbsuite.test.lldbtest import *
+from lldbsuite.test import lldbutil
 
-lldbinline.MakeInlineTest(__file__, globals(),
-                          decorators=[swiftTest,skipUnlessDarwin,
-                expectedFailureAll(bugnumber="rdar://60396797",
-                                   setting=('symbols.use-swift-clangimporter', 'false'))
-])
+
+class TestCase(TestBase):
+    @swiftTest
+    @skipUnlessDarwin
+    @expectedFailureAll(
+        bugnumber="rdar://60396797",
+        setting=("symbols.use-swift-clangimporter", "false"),
+    )
+    def test_measurement(self):
+        self.build()
+        lldbutil.run_to_source_breakpoint(
+            self, "break here", lldb.SBFileSpec("main.swift")
+        )
+        self.expect("expr -- measurement", substrs=["1.25 m"])
+
+    @swiftTest
+    @skipUnlessDarwin
+    def test_measurement_without_swift_ast_context(self):
+        self.build()
+        lldbutil.run_to_source_breakpoint(
+            self, "break here", lldb.SBFileSpec("main.swift")
+        )
+
+        # This setting is to test that measurements can be printed using
+        # TypeSystemSwiftTypeRef only. If the SwiftASTContext fallback were
+        # enabled, it can have the unwatned effect of suppressing failures
+        # within the TypeRef type system.
+        self.runCmd("settings set symbols.swift-enable-ast-context false")
+        self.addTearDownHook(
+            lambda: self.runCmd("settings set symbols.swift-enable-ast-context true")
+        )
```

Comment:
This used to be necessary, but dotest now just initialized a fresh lldb for each test.